run_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
needs header with more info, author, etc.

Reply:
We're doing this very inconsistently, currently. Do we want this? What do we want to be in there? We should standardize and document. But: I'll mimic what Thomas has done on his bot scripts for now.